metadata-ingestion/src/datahub/ingestion/source/sql/bigquery.py

```diff
@@ -835,14 +829,16 @@ def gen_schema_key(self, db_name: str, schema: str) -> PlatformKey:
             project_id=db_name,
             dataset_id=schema,
             platform=self.platform,
-            instance=self.config.env,
+            instance=self.config.platform_instance,
```

Comment:
My understanding is that we were going to just do this:

`instance=self.config.platform_instance if not None else self.config.env`

Reply:
WIthout adding any env to the key.